### PR TITLE
Skip old wince/winmo alpha builds of Fennec (fixes #224)

### LINKED ISSUE #224: Crash with Fennec wince
```
https://archive.mozilla.org/pub/mobile/releases/wince/1.0a3/fennec-1.0a3.en-US.wince-arm.zip
Traceback (most recent call last):
  File "/home/ubuntu/buildhub/jobs/buildhub/inventory_to_records.py", line 335, in csv_to_records
    record = record_from_url(url)
  File "/home/ubuntu/buildhub/jobs/buildhub/utils.py", line 314, in record_from_url
    "os": normalized_platform(platform),
  File "/home/ubuntu/buildhub/jobs/buildhub/utils.py", line 99, in normalized_platform
    raise ValueError("Unknown plaform {}".format(platform))
ValueError: Unknown plaform 1.0a3
```

### PR DESCRIPTION
**Pull request by leplatrem**
Fixes #224

### CODE REVIEW

File: jobs/buildhub/lambda_s3_inventory.py [hunk 1]
```diff
@@ -68,7 +68,7 @@
     Trigger to populate kinto with the last inventories.
     """
     server_url = os.getenv("SERVER_URL", "http://localhost:8888/v1")
-    bucket = os.getenv("BUCKET", "default")
+    bucket = os.getenv("BUCKET", "build-hub")
```
Comment (glasserc) [inline jobs/buildhub/lambda_s3_inventory.py]: Is this change related?

Comment (leplatrem) [inline jobs/buildhub/lambda_s3_inventory.py]: oops :| please excuse this
